ui/src/actions/__tests__/authors.test.js

```diff
@@ -96,6 +96,78 @@ describe('AUTHOR - async action creators', () => {
       done();
     });
 
+    it('and queries publications with proper size when user is cataloger', async done => {
```

Comment:
`but uses size=100 when user is cataloger and creates AUTHOR_PUBLICATIONS_SUCCESS if successful`